src/helpers/shared.ts

```diff
@@ -19,3 +19,15 @@ export const shouldSkip = (): { skip: boolean; reason: string } => {
 };
 
 export const wait = (ms: number) => new Promise((r) => setTimeout(r, ms));
+
+export const calculateWeight = (label: LabelItem | undefined): number => {
+  if (!label) return 0;
+  const matches = label.name.match(/\d+/);
+  const number = matches && matches.length > 0 ? parseInt(matches[0]) : 0;
+  if (label.name.split(":")[0] === "Priority") return number;
```

Comment:
```suggestion
  if (label.name.toLowerCase().includes("priority")) return number;
```
Maybe it's more flexible to check if it includes the word `priority`

Reply:
I am not sure what I have to do with time label weight...
@pavlovcik rfc.